### PR TITLE
LPS-164601 | Add Autom. test FDSTable#HeadersCanBeLocalized

### PR DESCRIPTION
**Pull request by deborasoliveira**
**Test Plan**

- Given: localized FDS sample portlet
- And Given: site URL localized to Spanish //localhost:8080/es/*
- When: on classic tab view
- Then: There is no hyphen in any header text
- And Then: First letter is capitalized for all headers
- And Then: localized Spanish headers are present //all headers have been translated to Spanish

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-164601

### CODE REVIEW

**Comment by deborasoliveira** [comment]
ci:test:sf

**Comment by deborasoliveira** [comment]
ci:test:frontend:nocompile

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSTable.testcase [hunk 1]
```diff
@@ -31,6 +31,10 @@ definition {
 
 			Navigator.gotoPage(pageName = "Frontend Data Set Test Page");
 		}
+
+		task ("And Given: site URL localized to Spanish //localhost:8080/es/*") {
+			Navigator.openSpecificURL(url = "http://localhost:8080/es/frontend-data-set-test-page/");
```
Comment (mateusralv) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSTable.testcase:36]: You can change the localhost to use a dynamic address, CI does not use port 8080, so the test is breaking.

```
var portalURL = PropsUtil.get("portal.url");

Navigator.openSpecificURLvalue1 = "${portalURL}es/frontend-data-set-test-page/");
```

Comment (mateusralv) [reply modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSTable.testcase:36]: And can you move this task for inside the test?

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSTable.testcase [hunk 2]
```diff
@@ -49,9 +53,35 @@ definition {
...
+				value1 = "Nombre");
+
+			AssertTextEquals.assertTextCaseSensitive(
+				key_columnName = "Apellido",
+				locator1 = "FrontendDataSet#COLUMN_NAME",
+				value1 = "Apellido");
+
+			AssertTextEquals.assertTextCaseSensitive(
+				key_columnName = "Dirección de correo",
+				locator1 = "FrontendDataSet#COLUMN_NAME",
```
Comment (mateusralv) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSTable.testcase:82]: Cool 🎉

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSTable.testcase (hunk 1 above)
Comment (deborasoliveira) [reply modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSTable.testcase:36]: okay!

**Comment by deborasoliveira** [comment]
ci:test:sf

**Comment by deborasoliveira** [comment]
ci:test:frontend:nocompile

**Comment by deborasoliveira** [comment]
ci:test:sf

**Comment by deborasoliveira** [comment]
ci:test:frontend:nocompile

**Comment by john-co** [comment]
ci:test:frontend-data-set:nocompile

**Comment by john-co** [comment]
hint to fix url: https://issues.liferay.com/browse/LRQA-77007

**Pull request closed without merging**